apps/src/templates/studioHomepages/ActionBlocksWrapper.jsx

```diff
@@ -0,0 +1,93 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {LinkButton} from '@cdo/apps/componentLibrary/button';
+import {
+  Heading2,
+  OverlineTwoText,
+  BodyThreeText,
+} from '@cdo/apps/componentLibrary/typography';
+import styles from './actionBlocksWrapper.module.scss';
+
+const OneColumnActionBlock = ({
+  overline,
+  imageUrl,
+  heading,
+  description,
+  buttons,
+}) => {
+  return (
+    <div className={styles.oneColumnActionBlock}>
+      <div className={styles.contentWrapper}>
+        {overline && (
+          <OverlineTwoText className={styles.overline}>
+            {overline}
+          </OverlineTwoText>
+        )}
+        {heading && (
+          <Heading2 visualAppearance="heading-md">{heading}</Heading2>
+        )}
+        {imageUrl && <img src={imageUrl} alt="" />}
+        {description && <BodyThreeText>{description}</BodyThreeText>}
+      </div>
+      <div className={styles.buttonWrapper}>
+        {buttons &&
+          buttons.map((button, index) => (
+            <LinkButton
+              key={index}
+              color={button.color}
+              href={button.url}
+              size="m"
+              text={button.text}
+              type={button.type}
+            />
+          ))}
+      </div>
+    </div>
+  );
+};
+
+OneColumnActionBlock.propTypes = {
+  overline: PropTypes.string,
```

Comment:
what, if any, of these are required out of curiousity? ie could this component be used without an image (for example)?

Reply:
Correct! Most will have at least a heading, image, description, and button, but I left off requirements to increase the flexibility. For example, we have some on Pegasus that just have a heading, description, and button.